styles/index.css

```diff
@@ -0,0 +1,29 @@
+body {
+    font-family: 'KoHo';
+    src: url('KoHo-Bold.woff2') format('woff2'),
+        url('KoHo-Bold.woff') format('woff');
+    font-weight: bold;
+    font-style: normal;
+    font-display: swap;
+}
+
+.sidebar {
+    background-color: beige;
+    width : 189px;
+    height: 38px;
+    padding: 20px;
+   
+}
+
+.sidebar ul {
+    padding: 0px;
+    list-style-type: none;
+}
+
+.sidebar ul li {
+    text-align: center;
+    padding: 8px;
+    font-size: 22px;
+    cursor: pointer; 
+    color: #211000;
```

Comment:
here we can use the variable, var(--textcolor-main);
To use it, make sure you link the variable stylesheet in html.

Reply:
ok